docs/user/integration/_sidebar.md

```diff
@@ -0,0 +1,6 @@
+- [Telemetry Module](../README.md)
```

Comment:
I understand the the nested sidebar can be deleted fully

Reply:
nice!